src/formatters/default_formatter.erl

```diff
@@ -1342,6 +1352,51 @@ lay_application(Name, Arguments, SpacesWithinParentheses, Ctxt) ->
             maybe_parentheses(D, Prec, Ctxt)
     end.
 
+%% @doc Might produce a new AST on which the functions in the export list
+%%      are sorted depending on what 'sort_function_exports' was set to:
+%%          - alphabetically, if set to 'alphabetically'
+%%          - left as it is, if set to 'false'
+sort_function_exports([Arguments0], SortFun) ->
+    Attrs = erl_syntax:get_pos(Arguments0),
+    case erl_anno:text(Attrs) of
+        %% If the attribute is indeed a export list, and the rule was enabled,
+        %% we sort the functions; otherwise, we ignore the attribute.
+        "export" ->
+            case erl_syntax:subtrees(Arguments0) of
```

Comment:
Can we use [`erl_syntax:attribute_arguments/1`](https://www.erlang.org/doc/man/erl_syntax.html#attribute_arguments-1) here?
Maybe something from [`erl_syntax_lib`](https://www.erlang.org/doc/man/erl_syntax_lib.html#analyze_export_attribute-1)?

Reply:
We could use stuff from `erl_syntax_lib` which would make our lives far easier if, in this point of the code, we still had the AST node and not its list of argument subtrees. E.g.:
```erlang
53> erl_syntax_lib:analyze_export_attribute(Node).
[{not_found,1},{no_updates,1}]
```

But that is not the case :/